### PR TITLE
kotlin 1.4.20

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> /usr/local/Cellar/kotlin/1.4.20/bin/kotlinc test.kt -include-runtime -d test.jar
The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com for information on installing Java.
```

File: Formula/kotlin.rb [hunk 1]
```diff
@@ -12,6 +12,8 @@ class Kotlin < Formula
 
   bottle :unneeded
 
+  depends_on "openjdk" => :test
```
Comment (SMillerDev) [inline Formula/kotlin.rb:15]: Seeing how it's Java based I'd say this is also at runtime.

Comment (chenrui333) [reply Formula/kotlin.rb:15]: yeah, that is what I thought (there gonna be some default java out for the runtime, but I am surprised to see the error)

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @bayandin!

**Pull request closed without merging**